README.md

```diff
@@ -6,7 +6,7 @@ External dependencies are linked via git submodules with an **ssh** link for eas
 
 # Requirements
 
-A Linux machine with modern multi-core cpus and at least 64GB RAM is recommend.
+A Linux machine with modern multi-core cpus and at least 16GB RAM is required.
```

Comment:
We can add 16GB RAM required, but I'd add 32GB RAM or more as heavily recommended for a good experience.

I think people running this on a 16GB RAM machine are bound to have an okish experience with this at best, especially if they have something else going on at the same time. I'd rather be upfront about it.

Reply:
Done.